virttest/utils_test/__init__.py

```diff
@@ -797,13 +797,12 @@ def transfer_data(session, host_cmd, guest_cmd, n_time, timeout,
 
     dir_name = data_dir.get_tmp_dir()
     transfer_timeout = int(params.get("transfer_timeout", 720))
-    tmp_dir = params.get("tmp_dir", data_dir.get_tmp_dir())
     filesize = int(params.get("filesize", 10))
     count = int(filesize)
 
     host_data_file = os.path.join(dir_name,
                                   "tmp-%s" % utils_misc.generate_random_string(8))
-    guest_data_file = os.path.join(tmp_dir,
+    guest_data_file = os.path.join(guest_path,
```

Comment:
`guest_path` is used for keeping the tool script file rather than the data file, it should use `tmp_dir` yet and just pick another default value to resolve the issue you met.

Reply:
@luckyh Okay, I will use `/var/tmp` instead, just not sure what do we use for windows guest in this case?

Reply:
Maybe `%TEMP%` I guess, @xiagao would you please help confirm this? thanks.